components/transfer.tsx

```diff
@@ -152,7 +152,7 @@ const Transfer = () => {
           symbol: "ZETA",
           formatted,
         })
-      } else if (sendType === "crossChainSwap") {
+      } else if (["crossChainSwap", "crossChainSwapBTC"].includes(sendType)) {
```

Comment:
nit: we could lowercase everything to avoid potential issues, as `sendType` has no type

Reply:
Good point. I've been getting away with not using types to speed up the dev process, but at some point I'll have to take care of this.